clients/java/src/main/java/io/zeebe/client/api/commands/ActivateJobsCommandStep1.java

```diff
@@ -86,5 +87,19 @@
      *     it to the broker.
      */
     ActivateJobsCommandStep3 workerName(String workerName);
+
+    /**
+     * Set a list of variable names which should be fetch on job activation.
+     *
+     * <p>The jobs which are activated by this command will only contain variables from this list in
+     * their payload.
+     *
+     * <p>This can be used to limit the number of variables in the payload of the activated jobs.
+     *
+     * @param fetchVariables list of variables names to fetch on activation
+     * @return the builder for this command. Call {@link #send()} to complete the command and send
+     *     it to the broker.
+     */
+    ActivateJobsCommandStep3 fetchVariables(List<String> fetchVariables);
```

Comment:
Can we add another method with a varargs parameter `fetchVariables(String... fetchVariables)`?

Reply:
I added vargs methods to activate jobs command and job worker builder, see commit 49509dfd1